packages/devtools_app/lib/src/scaffold.dart

```diff
@@ -402,8 +396,14 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
     final appBar = AppBar(
       // Turn off the appbar's back button.
       automaticallyImplyLeading: false,
-      title: Text(title),
+      title: Text(
+        title,
+        style: const TextStyle(
+          fontSize: 14,
```

Comment:
instead of setting the font size, can we use one of the text styles from TextTheme? headline2, headline3, etc.

Reply:
Done.